docs/content/02-tutorials/07-cost-tab.md

```diff
@@ -0,0 +1,56 @@
+
++++
+title = "f. Cost Tab 📈"
+weight = 27
++++
+
+This tutorial tells you how to view cluster spending costs and created budgets for clusters
```

Comment:
"This tutorial **shows** you how to view cluster costs broken down by instance type and create a budget for a specific cluster to get alerted when you approach your budget's cap."

Reply:
Resolved